kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -1,35 +1,347 @@
+                :style="accordionStyleOverrides"
+                class="accordion-header-label"
+                :aria-expanded="isItemExpanded(question.question_id)"
+                :aria-controls="`question-panel-${question.question_id}`"
+                @click="toggleItemState(question.question_id)"
+              >
+                <span>{{ title + " " + question.counter_in_exercise }}</span>
+                <KIcon
+                  style="position: absolute; right:1em; top: 0.625em;"
+                  :icon="isItemExpanded(question.question_id) ?
+                    'chevronUp' : 'chevronRight'"
+                />
+              </KButton>
+            </h3>
+          </template>
+          <template #content>
+            <div
+              v-if="isItemExpanded(question.question_id)"
+              :id="`question-panel-${question.question_id}`"
+              :ref="`question-panel-${question.question_id}`"
+              :style="{ userSelect: dragActive ? 'none!important' : 'text' }"
+            >
+              <ContentRenderer
+                :ref="`contentRenderer-${question.question_id}`"
+                :kind="activeResourceMap[question.exercise_id].kind"
+                :lang="activeResourceMap[question.exercise_id].lang"
+                :files="activeResourceMap[question.exercise_id].files"
+                :available="activeResourceMap[question.exercise_id].available"
+                :itemId="question.question_id"
+                :assessment="true"
+                :allowHints="false"
+                :interactive="false"
+                @interaction="() => null"
+                @updateProgress="() => null"
+                @updateContentState="() => null"
+                @error="err => $emit('error', err)"
+              />
+            </div>
+          </template>
+        </AccordionItem>
+      </template>
+    </AccordionContainer>
+    <div class="bottom-navigation">
+      <KGrid>
+        <KGridItem
+          style="text-align: left;"
+          :layout12="{ span: 8 }"
+          :layout8="{ span: 6 }"
+          :layout4="{ span: 3 }"
+        >
+          {{ numberOfSelectedReplacements$({
+            count: replacements.length,
+            total: selectedActiveQuestions.length }
+          )
+          }}
+        </KGridItem>
+        <KGridItem
+          style="text-align: right;"
+          :layout12="{ span: 4 }"
+          :layout8="{ span: 2 }"
+          :layout4="{ span: 1 }"
+        >
+          <KButton
+            :primary="true"
+            :text="replaceAction$()"
+            :disable="replacements.length !== selectedActiveQuestions.length"
+            @click="handleReplacement"
+          />
+        </KGridItem>
+      </KGrid>
+    </div>
+    <KModal
+      v-if="showCloseConfirmation"
+      :submitText="coreString('continueAction')"
+      :cancelText="coreString('cancelAction')"
+      :title="closeConfirmationTitle$()"
+      @cancel="handleCancelClose"
```

Comment:
One possibility instead of defining methods for these is to just define the handlers inline:
```
@cancel="showCloseConfirmation = false"
```

This might make it feel a _little_ cleaner?